rust/prql/src/lib.rs

```diff
@@ -0,0 +1,60 @@
+extern crate libc;
+
+use libc::{c_char, c_int};
+use prql_compiler::{json, prql_to_pl};
+use std::ffi::CStr;
+use std::ffi::CString;
+use prql_compiler::Options;
+
+#[no_mangle]
+pub unsafe extern "C" fn to_sql(query: *const c_char, out: *mut c_char) -> c_int {
+    let prql_query: String = CStr::from_ptr(query).to_string_lossy().into_owned();
```

Comment:
You may make this code simpler by using `cxx` crate, similar how `skim` is written right now.

Reply:
Actually this is really nice! I will use `cxx`, thanks!